web/styles/compose.css

```diff
@@ -1104,11 +1104,17 @@ textarea.new_message_textarea {
     color: var(--color-compose-recipient-box-text-color);
     background-color: var(--color-compose-recipient-box-background-color);
     border-color: var(--color-compose-recipient-box-border-color);
+    max-width: 30vw;
```

Comment:
@karlstolley FYI if you have a better approach in mind to show long stream names in compose at small widths so that topic name is not hidden.

Reply:
That approach probably works...not sure if it would be useful to set `width: max-content` above, or if that would create other problems.